.github/workflows/eden_rpi.yml

```diff
@@ -0,0 +1,140 @@
+---
+name: EdenRPi4
+on:  # yamllint disable-line rule:truthy
+  workflow_run:
+    workflows:
+      - Publish
+    types:
+      - completed
+# yamllint disable rule:line-length
+jobs:
+  integration:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        hv: ["kvm", "xen"]
+    steps:
+      - name: Check secrets
+        run: |
+          if [ -z "$ROL_API_URL" ]; then echo "::error::ROL_API_URL is empty" && exit 1; fi
+          if [ -z "$ROL_API_KEY" ]; then echo "::error::ROL_API_KEY is empty" && exit 1; fi
+          if [ -z "$ROL_PROJECT" ]; then echo "::error::ROL_PROJECT is empty" && exit 1; fi
+          if [ -z "$ROL_OVPN_CONF_BASE64" ]; then echo "::error::ROL_OVPN_CONF_BASE64 is empty" && exit 1; fi
+        env:
+          ROL_API_URL: ${{ secrets.ROL_API_URL }}
+          ROL_API_KEY: ${{ secrets.ROL_API_KEY }}
+          ROL_PROJECT: ${{ secrets.ROL_PROJECT }}
+          ROL_OVPN_CONF_BASE64: ${{ secrets.ROL_OVPN_CONF_BASE64 }}
+      - name: Get EVE
+        uses: actions/checkout@v2
+        with:
+          path: eve
+      - name: Setup packages
+        run: |
+          sudo apt update && sudo apt upgrade
+          sudo apt install -y curl golang net-tools qemu binfmt-support qemu-user-static qemu-utils qemu-system-x86 qemu-system-aarch64 openvpn jq
+      - name: Setup OVPN connection to RoL RPi4 demo
+        id: vpn
+        run: |
+          echo "$ROL_OVPN_CONF_BASE64" | base64 -d > ./config.ovpn
+          sudo openvpn --config ./config.ovpn --daemon
+          until ip -f inet addr show tun0; do sleep 5; ip a; done
+          echo ::set-output name=ip::$(ip -f inet addr show tun0 | sed -En -e 's/.*inet ([0-9.]+).*/\1/p')
+        env:
+          ROL_OVPN_CONF_BASE64: ${{ secrets.ROL_OVPN_CONF_BASE64 }}
+      - name: Prepare eden
+        run: |
+          if [ -f ${{ github.workspace }}/eve/tests/eden/eden-version ]; then
+            EDEN_VERSION=$(cat ${{ github.workspace }}/eve/tests/eden/eden-version)
+          else
+            EDEN_VERSION=lfedge/eden:30c8b3f
```

Comment:
I'd rather this be a release version instead of a weird sha

Reply:
i coppy this step from EdenGCP worckflow, this is not correct for us? 
https://github.com/lf-edge/eve/blob/master/.github/workflows/eden_gcp.yml#L62

Reply:
I use the EdenGCP workflow as a template, if we need to test the release versions only, that's not a problem to change, but I thought we wanted to test all the EVE images published to DockerHub i.e. snapshots too, this can be help determine problems more faster.

Reply:
Well, the fact that it is weird over there doesn't mean we should proliferate the weirdness. In fact @dautovri  should probably fix it over there as well.

Reply:
You can use recently deployed `lfedge/eden:0.3.0`

Reply:
Well, after pulling changes into Eden seems we should remove if-else logic here and use `lfedge/eden:0.4.0`, older versions have no support for command `eden rol`.

Reply:
yes, I am now using the 0.4.0 of EDEN tag.